tests/devices/unit_tests/test_eiger.py

```diff
@@ -53,6 +56,30 @@ def fake_eiger():
     return fake_eiger
 
 
+@pytest.fixture
+def finished_status():
+    status = Status()
+    status.set_finished()
+    return status
```

Comment:
Nit: I wouldn't make this a fixture, it can just be a function you call to get a status when you need one